Update using-the-visual-layer-with-xaml.md

vitordelucca opened the pull request:
It was InitializedFrostedGlass instead of InitializeFrostedGlass. So wouldn't work.

vitordelucca committed:
Update using-the-visual-layer-with-xaml.md

msftclas commented:
Hi __@vitorgrs__, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.microsoft.com.
    </span>

TTYL, MSBOT;

vitordelucca committed:
Update using-the-visual-layer-with-xaml.md

It was InitializedFrostedGlass instead of InitializeFrostedGlass, so it wouldn't work...

msftgits commented:
Hi, I am closing and re-opening this PR to bump the CLA bot. Sorry for the inconvenience!

msftclas commented:
Hi __@vitorgrs__, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, MSBOT;

mikematteson reviewed (approved):
Reviewed changes with content owner (Scott Miller); changes approved.
